src/unix/freebsd/disk.rs

```diff
@@ -166,19 +166,38 @@ impl GetValues for DiskInner {
     }
 }
 
-fn refresh_disk(disk: &mut DiskInner) -> bool {
+fn refresh_disk(disk: &mut DiskInner, refresh_kind: DiskRefreshKind) -> bool {
+    let (total_space, available_space) = if refresh_kind.details() {
     unsafe {
         let mut vfs: libc::statvfs = std::mem::zeroed();
         if libc::statvfs(disk.c_mount_point.as_ptr() as *const _, &mut vfs as *mut _) < 0 {
             return false;
         }
         let block_size: u64 = vfs.f_frsize as _;
 
-        disk.total_space = vfs.f_blocks.saturating_mul(block_size);
-        disk.available_space = vfs.f_favail.saturating_mul(block_size);
-        refresh_disk_io(&mut [disk]);
-        true
+        (
+            vfs.f_blocks.saturating_mul(block_size),
+            vfs.f_favail.saturating_mul(block_size),
+        )
     }
+    } else {
+        Default::default()
```

Comment:
Please use `(0, 0)` instead. Generated code with `Default::default()` is less good as you can see [here](https://godbolt.org/z/obMK6K1x4).

Reply:
How about `(Default::default(), Default::default())`? I know it's more verbose, but IMO it makes the intention much more legible to future maintainers.

Reply:
I'd prefer `(0, 0)` if you don't mind. At least I know when reading it that it's about integers.

Side note: `(Default::default(), Default::default())` is generating the same binary as `(0, 0)`. Always good to know that. :)

Reply:
Done!